orte/util/dash_host/dash_host.c

```diff
@@ -229,7 +229,7 @@ int orte_util_add_dash_host_nodes(opal_list_t *nodes,
                 found = true;
                 if (slots_given) {
                     node->slots += slots;
-                    if (0 < slots) {
+                    if (0 <= slots) {
```

Comment:
I would just remove the "if" statement as this change means it isn't doing anything.

Reply:
Take that back - I do see one place where slots is set to negative, so best to leave this change